pythonosc/osc_server.py

```diff
@@ -171,11 +171,19 @@ def datagram_received(self, data, unused_addr):
       _call_handlers_for_packet(data, self.dispatcher)
 
   def serve(self):
-    """creates a datagram endpoint and registers it with our event loop"""
+    """creates a datagram endpoint and registers it with our event loop.
+
+    Use this only if you are not currently running your asyncio loop.
+    (i.e. not from within a coroutine).
+    """
+    self._loop.run_until_complete(self.serve_async())
+
+  def serve_async(self):
```

Comment:
this doesn't effectively "serve" yet as the loop doesn't run yet, it just establishes the connection so please rename to something like "create_datagram_endpoint" to reuse the loop internal naming